framework/decode/dx12_replay_consumer_base.cpp

```diff
@@ -1060,24 +1082,32 @@ HRESULT Dx12ReplayConsumerBase::OverrideCreateCommandQueue(DxObjectInfo* replay_
 
     if (SUCCEEDED(replay_result))
     {
-        auto command_queue_info = std::make_unique<D3D12CommandQueueInfo>();
+        InitCommandQueueExtraInfo(replay_object, command_queue);
+    }
+
+    return replay_result;
+}
 
-        // Create the fence used for when command queues require sync after command list execution.
-        // Note that this fence is used only for waiting on command list execution. When needed, it will be signalled by
-        // the command queue in OverrideExecuteCommandLists and wait on in WaitForCommandListExecution.
-        auto fence_result =
-            replay_object->CreateFence(0, D3D12_FENCE_FLAG_NONE, IID_PPV_ARGS(&command_queue_info->sync_fence));
+HRESULT
+Dx12ReplayConsumerBase::OverrideCreateCommandQueue1(DxObjectInfo* device9_object_info,
+                                                    HRESULT       original_result,
+                                                    StructPointerDecoder<Decoded_D3D12_COMMAND_QUEUE_DESC>* desc,
+                                                    Decoded_GUID                                            creator_id,
+                                                    Decoded_GUID                                            riid,
+                                                    HandlePointerDecoder<void*>* command_queue_decoder)
+{
+    GFXRECON_UNREFERENCED_PARAMETER(original_result);
 
-        if (SUCCEEDED(fence_result))
-        {
-            command_queue_info->sync_event = CreateEventA(nullptr, TRUE, FALSE, nullptr);
-        }
-        else
-        {
-            GFXRECON_LOG_ERROR("Failed to create ID3D12Fence object used to sync after command list execution.");
-        }
+    assert((device9_object_info != nullptr) && (device9_object_info->object != nullptr) && (desc != nullptr) &&
```

Comment:
`GFXRECON_ASSERT`

The one in `OverrideCreateCommandQueue` could also use that mod

Reply:
Done